fastlane/Fastfile

```diff
@@ -771,6 +771,25 @@ lane :copyright do
   )
 end
 
+desc 'Get frameworks size'
+lane :get_frameworks_size do
+  xcodebuild(
+    archive: true,
+    archive_path: "Build/DemoApp.xcarchive",
+    scheme: "DemoApp"
+  )
+
+  stream_chat_size = File.size('../Build/DemoApp.xcarchive/Products/Library/Frameworks/ChatSample.app/Frameworks/StreamChat.framework/StreamChat')
+  stream_chat_ui_size = File.size('../Build/DemoApp.xcarchive/Products/Library/Frameworks/ChatSample.app/Frameworks/StreamChatUI.framework/StreamChatUI')
+
+  # Converting to MB, not MiB (1024 bytes), since macOS finder uses MB
+  stream_chat_size_in_mb = (stream_chat_size.to_f / 1000 / 1000).round(2)
+  stream_chat_ui_size_in_mb = (stream_chat_ui_size.to_f / 1000 / 1000).round(2)
+  
+  UI.success("StreamChat: #{stream_chat_size_in_mb}MB")
+  UI.success("StreamChatUI: #{stream_chat_ui_size_in_mb}MB")
+end
```

Comment:
I would be so nice to post results to PR. Otherwise it is too hidden.

Reply:
Yes, right now, this is a manual process. Eventually, we would like to do one or all of these, probably a task for @testableapple:
- Run this for every release and update the README size labels
- Run this for every PR (3-4min increase on CI because of the archive, but maybe there's way to optimize) and show the difference from the previous release/commit (The diff can be a simple subtract of the new and old size)
- The latest stage would be to run it for every PR and show the difference at a file/source code level using [bloaty](https://github.com/google/bloaty)

Reply:
Yep, sounds like a plan. Let's get the informative piece in now.

Reply:
Wow that's awesome! It will be great it bring it over also to video. Nice job!